projects/components/src/menu-dropdown/menu-item/menu-item.component.ts

```diff
@@ -15,7 +15,7 @@ import { IconSize } from '../../icon/icon-size';
         size="${IconSize.Small}"
         [color]="this.iconColor"
       ></ht-icon>
-      <span class="label">{{ this.label }}</span>
+      <span class="label" [ngStyle]="{ color: this.labelColor ? this.labelColor : '' }">{{ this.label }}</span>
```

Comment:
If labelColor is not set, does this affect the inherited color (I don't think so, just want to verify)

Reply:
I checked and it is not affecting. In case of not passing a variable to it, it keeps the previous value.